action/protocol/staking/protocol.go

```diff
@@ -95,6 +96,11 @@ type (
 		PersistStakingPatchBlock         uint64
 		EndorsementWithdrawWaitingBlocks uint64
 	}
+	// HelperCtx is the helper context for staking protocol
+	HelperCtx struct {
+		GetBlockInterval func(uint64) time.Duration
```

Comment:
where is new funcs being used?

Reply:
It will be used at #4299 to calculate the migrated nft bucket duration in block num unit